src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/TstInfo.cs

```diff
@@ -160,4 +160,4 @@ internal static TstInfo Read(DerSequenceReader reader)
                 extensions);
```

Comment:
Undo changes to this file.

Reply:
Since `insert_final_newline = true` in NuGet.Client/.editorconfig. It adds a new line by default when I save it.

Reply:
Undo it.  Don't save it.

Reply:
@heng-liu feel free to ping me if you need help removing specific changes from git

Reply:
@zkat Thanks a lot for your help offline! :)